server/src/main/java/org/elasticsearch/cluster/metadata/IndexTemplateMetadata.java

```diff
@@ -161,6 +161,11 @@ public static Builder builder(String name) {
         return new Builder(name);
     }
 
+    @Override
+    public String toString() {
```

Comment:
Convenience method so that the errors we return actual contain the template name we failed on.